### PR TITLE
LocalDev - UI solution

### PR DESCRIPTION
**Pull request by sewerynmi**
Removed temp and not required files

### CODE REVIEW

**Review by stephenkaye-ho — APPROVED** [review]
Looks. good

File: README.md [hunk 1]
```diff
@@ -130,3 +130,32 @@ VITE_ACCRUALS_API_URL_PROXY="http://localhost:9091/"
...
 
 If you’re using an AMD (M1,M2) chip mac, you might encounter problems starting the keycloak docker container. We’ve had some success building a new image locally, as per the instructions in this github thread: https://github.com/docker/for-mac/issues/5310#issuecomment-877653653
+
+## Running UI in LocalDev Environment
+
+### Step 1
+
+Download Callisto LocalDev repository from https://github.com/UKHomeOffice/callisto-localdev
+
+Run its services with command : `docker compose up -d`
```
Comment (stephenkaye-ho) [inline README.md:140]: I think this should be changed to tell them to follow the instructions in the callisto-localdev readme

File: README.md [hunk 2]
```diff
@@ -130,3 +130,32 @@ VITE_ACCRUALS_API_URL_PROXY="http://localhost:9091/"
...
+
+### Step 2
+
+Stop (manually) service `web` (most likely the name will be `web-1`)
```
Comment (stephenkaye-ho) [inline README.md:144]: We should change this to use the docker compose method
`docker compose stop web`

File: README.md [hunk 3]
```diff
@@ -130,3 +130,32 @@ VITE_ACCRUALS_API_URL_PROXY="http://localhost:9091/"
...
+
+### Step 3
+
+Create file .env in the root of this project, and add the following:
```
Comment (stephenkaye-ho) [inline README.md:148]: If this is needed then we should check in the .env file with these settings and remove it from the git ignore. The aim here is zero config. If localdev is in use, these values wont change. If we are concerned about conflicting with current development not using docker compose (localdev) then we should use a different file name e.g. docker.env and then use the [env_file configuration option](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option)

**Review by stephenkaye-ho — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sewerynmi) [reply README.md:144]: sure, let's use docker compose for that
Fixed.

File: README.md (hunk 3 above)
Comment (sewerynmi) [reply README.md:148]: I guess I will change it to use 2 files:  .env and .env.local
We're going to checkout .env but not .env.local which will be specific for each developer. 
LocalDev variables will be stored in the .env file. This one shouldn't be edited by developers. 
Variables for a solution without using LocalDev will be stored in the .env.local which takes precedence over the .env file. Developers can do anything they want in that file. For example: enabling the usage of mock data etc. 
Is that ok?

Comment (stephenkaye-ho) [reply README.md:148]: So this bit won't be needed, we'll check in the .env file with correct settings and remove it from the .gitignore file. I don't know if we need to document the .env.local here or if that's just a common thing to document elsewhere. I presume it's taking advantaged of docker using .env whilst the react app falls back from .env.local to .env allowing us to use it as an override/alternative

File: package.json [hunk 4]
```diff
@@ -4,17 +4,19 @@
...
+    "start:jsonserver": "node mocks/apiServer.js",
     "test": "TZ=UTC jest --coverage",
     "monitor": "TZ=UTC jest --watchAll",
     "lint:fix": "eslint --ext .js,.jsx src --fix",
     "lint": "eslint --ext .js,.jsx src",
     "build": "vite build",
     "preview": "vite preview",
-    "vite": "vite"
+    "vite": "vite",
+    "localdev": "run-p localdev:init start:jsonserver vite ",
```
Comment (stephenkaye-ho) [inline package.json:18]: should this be `run-p localdev jsonserver vite`
and we should be using pre/post prefixes for the init. Should we always start the jsonserver (that's probably a sensible choice). I think we should tidy up these scripts and drop out keycloak. The timezone env var is probably a good candidate for also going in the .env file

File: package.json [hunk 5]
```diff
@@ -4,17 +4,19 @@
...
+    "localdev:init": "npm install"
```
Comment (stephenkaye-ho) [inline package.json:19]: When is this used?

**Review by stephenkaye-ho — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (stephenkaye-ho) [inline package.json:18]: this also fails for me on a clean build because 
```
sh: run-p: not found
```

Comment (stephenkaye-ho) [reply package.json]: Realised some other issue with the npm script already depending on run-p before npm install is guaranteed to have been called so moved this into the entry point as a shell command. This means there are no changes to the script section so the configuration/structure can be assessed elsewhere as this PR no longer touches them

**Review by stephenkaye-ho — APPROVED** [review]

**Pull request merged**